crates/rspack_plugin_javascript/src/visitors/dependency/scanner.rs

```diff
@@ -504,6 +505,13 @@ fn scanner_context_module(expr: &Expr) -> Option<(String, String)> {
   }
 }
 
+#[inline]
+fn quote_meta(str: String) -> String {
+  let re =
+    Regex::new(r"[-\[\]\\/{}()*+?.^$|]").expect("Failed to initialize `metacharacters REGEXP`");
```

Comment:
The regex instance can be static use `Lazy::new`.

Reply:
ok